databases/week1/homework.sql

```diff
@@ -0,0 +1,103 @@
+Select
+*
+FROM
+task;
+
+-- 1.Find out how many tasks are in the task table
+SELECT COUNT(id)
```

Comment:
COUNT(task.id) as task_Count
_Pointer_
Add column name for columns you have created